parser/worker.go

```diff
@@ -60,7 +61,9 @@ func (w Worker) Start() {
 
 	for i := range w.queue {
 		if err := w.ProcessIfNotExists(i); err != nil {
-			// re-enqueue any failed job
+			// re-enqueue any failed job after average block time
+			time.Sleep(config.Cfg.Parser.AvgBlockTime)
```

Comment:
I'm curious about what happens if this is not defined (the user has not specified an `average_block_time` inside their configuration). Will this throw some error or what? Should be consider that case and handle it without failing?